Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -171,14 +174,17 @@ else {
 	println "*** Number of build outputs to publish: $loadCount"
 	loadDatasetToMembersMap.each { dataset, members ->
 		members.each { member ->
-
+			def deployType = dataset.replaceAll(/.*\.([^.]*)/, "\$1")
```

Comment:
I have a hard time to understand and relate the variable name `deployType` to the way you it in the updated script. 

The way I understand your contribution, your are obtaining the last-level qualifier, which is (optionally) added as a file extension. 

While `deployType` is an (optional) attribute on the DBB Execute record for the build outputs, this will be confusing to users of this script.

Should we allow the user to specify a value of the cli argument `addExtension` to determine if they would like to have the DBB DeployType or the LLQ?

This might require additional updates to the script, while the DBB build results record is "lost" during the processing flow.